Document WebdriverIO basic setup

clample opened the pull request.

Commit:
begin WebdriverIO basic setup docs

ragog commented on docs/tools/appium/setups/webdriverio/setup-webdriverio-basic.md:
```diff
@@ -0,0 +1,93 @@
+});
+{% endhighlight %}
+
+Now that all of the neccessary files have been created, it is possible to run the tests. First, install the dependencies by running `npm install`. Next, run the tests by running `npm test`.
```
"necessary"

ragog commented on docs/tools/appium/setups-landing.md:
```diff
@@ -35,6 +35,12 @@ Ruby is our most recent addition as for supported languages for Appium tests. As
 </ul>
 
+<h4 id="webdriverio">WebdriverIO</h4>
+WebdriverIO is a testing framework allowing you to write webtests using Javascript. Currently only the basic setup is supported.
```
"web tests" I guess

ragog replied:
also expand by saying "Currently only the basic setup is supported. This means the result of your test won't be saved in your test report on TestObject." or so.

ragog commented on docs/tools/appium/setups/webdriverio/setup-webdriverio-basic.md:
```diff
@@ -0,0 +1,93 @@
+permalink: docs/tools/appium/setups/webdriverio/basic/
+---
+
+<a href="http://webdriver.io" target="_blank">WebdriverIO</a> is a powerful framework for webtesting. Webtests written using WebdriverIO can easily be configured to run on real mobile devices on TestObject. You can find a working example for running WebdriverIO tests on our github page: <a href="https://github.com/testobject/webdriverio-example" target="_blank">TestObject WebdriverIO example</a>.
```
Again, wouldn't make "webtest" a single word, but you are the native speaker so it's your call ;)

Commit:
Fix typos and add clarification

- It makes sense to give more information on what it means that only a
  basic setup is supported. This will help users to decide if they want
  to use this framework for testing
